src/sage/rings/function_field/element.pyx

```diff
@@ -174,6 +178,260 @@ cdef class FunctionFieldElement(FieldElement):
         """
         return self._x._latex_()
 
+    def subs(self, in_dict=None, **kwds):
+        r"""
+        Substitutes given generators with given values while not touching
```

Comment:
```suggestion
        Substitute the given generators with given values while not touching
```

Reply:
Thanks, but that is (very) old documentation before we have our current standards. It would be good to change that one. Independently, consistency is not so important.